test/options/generatereadmemd-vanilla-true/Expected/AcceptanceTests/ParameterFlattening/README.md

```diff
@@ -3,6 +3,7 @@ uid: azure-arm-parameterflattening
 summary: *content
 
 ---
+**This package will eventually be replaced by our new TypeScript-based isomorphic SDKs. Learn more at the [GitHub repository for our new packages](https://github.com/Azure/azure-sdk-for-js).**
 # Microsoft Azure SDK for Node.js - AutoRestParameterFlattening

```

Comment:
@dend - If this is changed from `#` to `##`, then I assume this will solve the problem of conflicting Heading in README.md Then clicking on a package name in the left menu bar should show the content of README as opposed to nothing (that is shown today), correct?

Reply:
Yes, that will be helpful for us, because the first heading is generally reserved for the package name.